app/dashboard/views/unsubscribe.py

```diff
@@ -68,3 +70,46 @@ def block_contact(contact_id):
         )
     else:  # ask user confirmation
         return render_template("dashboard/block_contact.html", contact=contact)
+
+
+@dashboard_bp.route("/encoded", methods=["GET"])
```

Comment:
can we make the url more explicit, for example `/encoded-unsubscribe` instead of just `/encoded`?

Reply:
Will do.

Reply:
On second thought. The route for this will be `/dashboard/unsubscribe/encoded` as it is. Maybe we don't need to add another `unsubscribe` at the end?

Reply:
Yep. I was wrong. Fixed. Not its `/dashboard/unsubscribe/encoded/<request>`